src/chrome/content/rules/GeForce.com.xml

```diff
@@ -1,51 +1,28 @@
 <!--
 	For other nVidia coverage, see NVidia.xml.
```

Comment:
There is a bunch of language subdomains that does not match the given certificate, like `cn.geforce.com`, `fr.geforce.com`, `uk.geforce.com` etc.

Reply:
ping @koops76

Reply:
@J0WI Done.